apps/homework/models.py

```diff
@@ -86,9 +88,10 @@ def get_challenge_url(self):
         if not self.definition.challenge_url:
             print("No challenge URL given.")
             return
-        if not self.github_url:
-            print("No submission github URL given.")
-            return
+        # Pulled for now, as with direct file upload it's possible to not be given a github_url
```

Comment:
cool, good comment, should we just pull this all together?

Reply:
👍